plugins/notifications-backend/src/service/handlers.ts

```diff
@@ -95,14 +104,31 @@ export function getNotifications(
         title: message.title,
         message: message.message,
         topic: message.topic,
+        actions: [],
       };
       return notification;
+    }),
+  );
+
+  const actionsMessageIds = notifications.map(notification => notification.id);
+
+  const actionsQuery = dbClient('actions')
+    .select('*')
+    .whereIn('messages_id', actionsMessageIds);
```

Comment:
message_id

Reply:
I am more used to `[table]_[column]` pattern but it's not so important. Changed.